MonoGame.Framework/Audio/Xact/SoundBank.cs

```diff
@@ -93,125 +94,132 @@ public SoundBank(AudioEngine audioEngine, string fileName)
                     string[] cueNames = System.Text.Encoding.UTF8.GetString(reader.ReadBytes((int)cueNameTableLen), 0, (int)cueNameTableLen).Split('\0');
 
                     // Simple cues
-					stream.Seek(simpleCuesOffset, SeekOrigin.Begin);
-					for (int i=0; i<numSimpleCues; i++) 
+                    if (numSimpleCues > 0)
                     {
-                        reader.ReadByte(); // flags
-                        uint soundOffset = reader.ReadUInt32();
-
-                        var oldPosition = stream.Position;
-                        stream.Seek(soundOffset, SeekOrigin.Begin);
-						XactSound sound = new XactSound(this, reader);
-                        stream.Seek(oldPosition, SeekOrigin.Begin);
-
-                        Cue cue = new Cue(_audioengine, cueNames[i], sound);
-                        _cues.Add(cue.Name, cue);
-					}
-					
-                    // Complex cues
-                    stream.Seek(complexCuesOffset, SeekOrigin.Begin);
-					for (int i=0; i<numComplexCues; i++) 
-                    {
-						Cue cue;
-						
-						byte flags = reader.ReadByte ();
-						if (((flags >> 2) & 1) != 0) 
+                        stream.Seek(simpleCuesOffset, SeekOrigin.Begin);
+                        for (int i = 0; i < numSimpleCues; i++)
                         {
+                            reader.ReadByte(); // flags
                             uint soundOffset = reader.ReadUInt32();
-                            reader.ReadUInt32(); //unkn
 
                             var oldPosition = stream.Position;
                             stream.Seek(soundOffset, SeekOrigin.Begin);
                             XactSound sound = new XactSound(this, reader);
                             stream.Seek(oldPosition, SeekOrigin.Begin);
 
-                            cue = new Cue(_audioengine, cueNames[numSimpleCues + i], sound);
-						} 
-                        else 
+                            Cue cue = new Cue(_audioengine, cueNames[i], sound);
+                            _cues.Add(cue.Name, cue);
+                        }
+                    }
+					
+                    // Complex cues
+                    if (numComplexCues)
```

Comment:
It looks like there was a merge problem as I moved the code around and I must have retyped that line by accident. I'll fix the branch and commit that.